api/lib/domain/models/CertifiedLevel.js

```diff
@@ -16,63 +16,63 @@ class CertifiedLevel {
     reproducibilityRate,
   }) {
     if (numberOfChallengesAnswered === 3) {
-      return this._scoreFor3Challenges({ numberOfCorrectAnswers, estimatedLevel, reproducibilityRate });
+      return CertifiedLevel._for3Challenges({ numberOfCorrectAnswers, estimatedLevel, reproducibilityRate });
```

Comment:
Quelle est la différence avec `this._for3challenges` ?

Reply:
Répondu à l'oral : à l'exécution aucune (ça marche pareil) c'est plus facile de comprendre qu'on parle d'un méthode statique si on fait `Classe.methode()` plutôt que `this.méthode()`, `this` se référent le plus souvent à une instance. Dans certain langage on pourrait avoir la différence en `self` et `this` pour faire ce distinguo mais pas en JS donc on part sur `Classe.méthode()`.